toolset/utils/docker_helper.py

```diff
@@ -162,6 +164,15 @@ def watch_container(container):
             print(e)
             return 1
 
+    return 0
+
+
+def successfully_running_containers(docker_files, database_container_id, out):
```

Comment:
```python
if (running_container_length != expected_length):
```

I'm wondering about the value of `!=` here as opposed to `<`.  Recent runs in the Citrine environment were failing all frameworks that used a database with errors like this:

```
Running Containers (id, name):
b5a97fbe02, <Image: 'tfb/test/undertow-mysql:latest'>
Excepted 1 running containers; saw 2
ERROR: Problem starting undertow-mysql
```

The theory was that a container was sticking around from a previous test?  If nothing has changed to make that situation less likely, then I think `<` would be better than `!=` if only for the sake of the Citrine environment.

Reply:
Specifically, if we do not have exactly the expected number of running containers then things have gone wrong and we want to error on that.

I expect containers to be properly stopped and that if we're expecting 1 running container, then we will only see 1 running container; if that is not true, we need to address that directly.